.github/workflows/build_and_test.yml

```diff
@@ -412,6 +412,30 @@ jobs:
         shell: bash
         run: unset RUSTC && ARCH=${{ matrix.arch }} RUN_ON_CI=1 LLVM_CONFIG=llvm-config-${{env.MAIN_LLVM_VERSION}} cargo run --manifest-path ./utils/ci_runner/Cargo.toml --  ${{ matrix.fuzzer }}
 
+  utils:
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
+        fuzzer:
+          # Binary only
+          - ./utils/gdb_qemu
+        arch:
+          # unless somebody pays us for the servers.
+          # - aarch64
+          # - arm
+          # - i386
+          # - ppc
+          - x86_64
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/workflows/ubuntu-prepare
+      - name: Build and run utils (Linux)
+        if: runner.os == 'Linux'
+        shell: bash
+        run: unset RUSTC && ARCH=${{ matrix.arch }} RUN_ON_CI=1 LLVM_CONFIG=llvm-config-${{env.MAIN_LLVM_VERSION}} cargo run --manifest-path ./utils/ci_runner/Cargo.toml --  ${{ matrix.fuzzer }}
```

Comment:
Can't we just call the justfile here? Not sure we need the CI runner as extra indirection(?)

Reply:
Hopefully this looks better.